spesifikasjoner/afpant/afpant-kjoepekontrakt/kjoepekontrakt-loesningsbeskrivelse.md

```diff
@@ -54,6 +61,15 @@ Signert kjøpekontrakt skal alltid sendes elektronisk, uavhengig av om det er el
 
 Meldingstype: [KjoepekontraktSvarFraMegler](./kjoepekontrakt-teknisk-beskrivelse.md#meldingstype-kjoepekontraktsvarframegler)
 
+## Oversendt informasjon er endret
+Det er viktig at banken har oppdatert informasjon i systemene og at denne er lik det som ligger i det signerte dokumentet. 
+Dersom megler endrer eller legger til informasjon i meglersystemene, må banken få beskjed om dette. Det gjøres ved at 
+meglersystemet sender en [melding](./kjoepekontrakt-teknisk-beskrivelse.md#meldingstype-kjoepekontraktframegler) til banken. 
+
+Hvilke felter som skal trigge en melding til banken om at det har skjedd en endring, er merket i tabellen under [Svar fra megler](#svar-fra-megler).
+
+Meldingstype: [KjoepekontraktFraMegler](./kjoepekontrakt-teknisk-beskrivelse.md#meldingstype-kjoepekontraktframegler)
+
```

Comment:
Vi har introdusert to "submeldingstyper" her i xsd, med hhv name kjoepekontraktEndringFraMegler og kjoepekontraktUoppfordretFraMegler. Begge er har type KjoepekontraktFraMegler i xsd'en forøvrig. Hva tenker du her @ropatambitadotcom, bør vi splitte opp i dokumentasjonen det linkes til her eller tenker vi det er forløpig greit å bare ha KjoepekontraktFraMegler inntil videre evt?